lib/components.rb

```diff
@@ -128,7 +128,9 @@ def self.setup
         repo: "atlantis-builder",
         preimage: lambda do |component|
           FileUtils.cp "#{ENV["HOME"]}/.ssh/id_rsa.pub", component.directory
+          status = Status.read
```

Comment:
not sure... was following what other component does. not sure what it does, if anything.